notify/impl.go

```diff
@@ -813,3 +821,30 @@ func tmplHTML(tmpl *template.Template, data *template.Data, err *error) func(str
 		return s
 	}
 }
+
+type loginAuth struct {
+  username, password string
+}
+
+func LoginAuth(username, password string) smtp.Auth {
+    return &loginAuth{username, password}
+}
+
+func (a *loginAuth) Start(server *smtp.ServerInfo) (string, []byte, error) {
+    return "LOGIN", []byte{}, nil
+}
+
+// Used for AUTH LOGIN. (Add base64 encrypt mode).
+func (a *loginAuth) Next(fromServer []byte, more bool) ([]byte, error) {
+    if more {
+        switch string(fromServer) {
+        case "Username:":
+                return base64.StdEncoding.DecodeString(a.username)
```

Comment:
I think in the future, it will be good to encrypt all username/password. Because it's no secured to add some passwords in a config file without encrypted their.
Why do you think about ?